powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/v3/PowerAuthController.java

```diff
@@ -497,4 +497,28 @@ public void setPowerAuthService(PowerAuthService powerAuthService) {
         return new RESTResponseWrapper<>("OK", powerAuthService.removeToken(request.getRequestObject()));
     }
 
+    /**
+     * Call {@link PowerAuthService#startMigration(StartMigrationRequest)} method and
+     * return the response.
+     * @param request Start migration request.
+     * @return Start migration response.
+     * @throws Exception In case the service throws exception.
+     */
+    @RequestMapping(value = "/migration/start", method = RequestMethod.POST)
```

Comment:
`/activation/migration/start` - migration relates to activations...

Just out of curiosity, why do we "migrate" activations (and where do we migrate them?) instead of "upgrading them" (as to new version)?

Reply:
As agreed we will use `upgrade` instead of `migration`.